decision-records/DR-0001-why-docusaurus.md

```diff
@@ -0,0 +1,115 @@
+# Why we use Docusaurus
+
+| Status      | implemented                                                                             |
+| :---------- | :-------------------------------------------------------------------------------------- |
+| **PR #**    | [2](https://github.com/blindnet-io/blindnet-docs/issues/2)                              |
+| **Author**  | Marko Stakic (marko@blindnet.io)                                                        |
+| **Reviwer** | Noël Macé (noel@blindnet.io)                                                            |
```

Comment:
```suggestion
| **Author** & **Sponsor** | Marko Stakic (marko@blindnet.io)                                                        |
| **Lead Reviewer** | Noël Macé (noel@blindnet.io)                                                            |
```

See https://github.com/blindnet-io/openness-framework/issues/14 about the sponsor part.